plugin-guanyin/src/main/java/ca/on/oicr/gsi/shesmu/guanyin/RunReport.java

```diff
@@ -268,12 +266,18 @@ public ActionState perform(ActionServices services) {
                   "1.0",
                   MAPPER.writeValueAsString(labels),
                   null);
+          this.errors = Collections.emptyList();
         } else if (cromwellId != null) {
           cromwellId = wfApi.status("v1", cromwellId.getId());
+          this.errors = Collections.emptyList();
+        } else {
+          this.errors =
+              Collections.singletonList("Report has already been launched but ID is unknown.");
```

Comment:
Is it the case that this may resolve with time? It's not clear if user action is required, or what user action could be performed.